src/delay.js

```diff
@@ -299,4 +298,15 @@ p5.Delay.prototype.dispose = function () {
   this.leftDelay = undefined;
   this.rightDelay = undefined;
 };
-export default p5.Delay;
+ 
+};
+
+
+
+
+
+
+
+
+
```

Comment:
all these blank lines are unnecessary (another thing I wish the linter caught!)

Reply:
I think i should run the linter again
Will be reporting it soon